solutions/USACO dec19-milkvisits-silver.mdx

```diff
@@ -2,13 +2,46 @@
 id: usaco-968
 source: USACO Silver 2019 December
 title: Milk Visits
-author: Qi Wang
+author: Qi Wang, Tanish Tyagi
 ---
 
 [Official Analysis](http://www.usaco.org/current/data/sol_milkvisits_silver_dec19.html)
 
 **Time Complexity:** $\mathcal{O}(N + M)$
 
+We can use a DFS to create connected components from an adjacency list (which is made when we process the roads between the farms). Each connected component will contain farms that all have the same breed. 
+
+Adjacency List based off sample test case: <br/>
+1 --> 2, 5  <br/>
```

Comment:
can u add latex to this section? personally i think it looks better with latex.

\rightarrow for the arrow btw.